packages.json

```
@@ -16809,5 +16810,20 @@
     "license": "MIT",
     "web": "https://github.com/bunkford/winserial",
     "doc": "https://bunkford.github.io/winserial/winserial.html"
+  },
+  {
+    "name": "nimbler",
+    "url": "https://github.com/paul-nameless/nimbler",
+    "method": "git",
+    "tags": [
+      "web",
+      "http",
+      "rest",
+      "api",
+      "library"
+    ],
+    "description": "A new awesome nimble package",
```

You might want to edit this, so that its a bit more descriptive, maybe lke this:
```suggestion
    "description": "A library to help you write rest APIs",
```